### PR TITLE
Update makefile and specfile

### PR DESCRIPTION
**Pull request by RobSanders**

### CODE REVIEW

**Comment by RobSanders** [comment]
- Updated Makefile rpm and clean targets
- Bumped version on spec file to match Makefile
- Added changelog entries for my just merged commits
- Fixed dates on existing changelog entries to fix rpmbuild errors/warnings
- Updates spec file similar to EPEL's version to generate regular and devel packages

File: libcli.spec [hunk 1]
```diff
@@ -8,35 +8,75 @@ Source: %{name}-%{version}.tar.gz
...
 
 %install
 rm -rf $RPM_BUILD_ROOT
-make DESTDIR=$RPM_BUILD_ROOT PREFIX=/usr install
-find $RPM_BUILD_ROOT/usr ! -type d -print | grep -v '\/(README|\.html)$' | \
-    sed "s@^$RPM_BUILD_ROOT@@g" | sed "s/^\(.*\)$/\1\*/" > %{name}-%{version}-filelist
+install -d -p %{buildroot}%{_includedir}
+install -p -m 644 libcli*.h %{buildroot}%{_includedir}/
+install -d -p %{buildroot}%{_libdir}
+install -p -m 755 libcli.so.1.9.8 %{buildroot}%{_libdir}/
```
Comment (dparrish) [inline libcli.spec]: Can this not use %{version} here instead of hardcoding the version number?

File: libcli.spec [hunk 2]
```diff
@@ -8,35 +8,75 @@ Source: %{name}-%{version}.tar.gz
...
+* Tue Sep 18 2018 Rob Sanders <rsanders@forcepoint.com> 1.9.8-3
+- Update spec file similar to EPEL's for regular and devel pacakges
+- Update Makefile rpm target to build both regular and devel pacakges
+- Update changelog for new fixes)
+- Update changelog (fix dates on several commits to avoid rpmbuild complaint)
+
+* Sun Sep 16 2018 David Parrish <david@parrish.com> 1.9.8-2
+- Reformat patches with clang-format
+
+* Thu Sep 13 2018 Rob Sanders <rsanders.forcepoint.com> 1.9.8-1
```
Comment (dparrish) [inline libcli.spec]: rsanders@forcepoint.com

**Review by dparrish — COMMENTED** [review]
Thank you very much!

File: libcli.spec (hunk 1 above)
Comment (RobSanders) [reply libcli.spec]: Probably, I was thinking down those lines, additionally for the symbolic links for the libraries just a few lines down.   Would need to truncate the last digit as well.

File: libcli.spec (hunk 2 above)
Comment (RobSanders) [reply libcli.spec]: Ooops....good catch.

**Pull request merged**